packages/next/src/client/components/router-reducer/is-prefetch-cache-entry-expired.ts

```diff
@@ -0,0 +1,17 @@
+import { PrefetchCacheEntry } from './router-reducer-types'
+
+const FIVE_MINUTES = 5 * 60 * 1000
+const THIRTY_SECONDS = 30 * 1000
+
+export function isPrefetchCacheEntryExpired(
+  cacheEntry: PrefetchCacheEntry
+): boolean {
+  return cacheEntry?.kind === 'hard'
+    ? // check if the prefetchTime was less than 5 mins ago
+      Date.now() < cacheEntry.prefetchTime + FIVE_MINUTES && // check if the lastUsedTime was less than 30s ago or if it was never used
+        (!cacheEntry.lastUsedTime ||
+          Date.now() < cacheEntry.lastUsedTime + THIRTY_SECONDS)
+    : // check if the lastUsedTime was less than 30s ago or if it was never used
+      !cacheEntry.lastUsedTime ||
+        Date.now() < cacheEntry.lastUsedTime + THIRTY_SECONDS
```

Comment:
Shouldn't these conditions be `Date.now() > cacheEntry.prefetchTime + FIVE_MINUTES` and `Date.now() > cacheEntry.lastUsedTime + THIRTY_SECONDS` to check if the cache entry has expired ?

Because with this, how i read this : once the cache has been prefetched more than 5 minutes ago (or used more than 30s ago), the cache is considered fresh which i think does not make sense. 

maybe i am wrong and if it is the case, sorry for bothering you.